src/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const PAGINATION_TYPE = {
+    PAGINATION: 'page',
+    CONTINUATION: 'continuation',
+};
+
+export const PAGINATION_MAP = {
```

Comment:
We are currently using this internally, I think we can abstract it a bit and add it here. I think we may need to address some of the terms, but I wanted thoughts before I went too far down this path.